inspirehep/dojson/common/base.py

```diff
@@ -216,10 +217,13 @@ def collections(self, key, value):
 
     def get_value(value):
```

Comment:
Not a fan of defining this function every time this rule is applied — but also not sure if there are performance issues! @mihaibivol, do you know if this hurts performance?